keras_cv/layers/preprocessing/with_mixed_precision_test.py

```diff
@@ -30,6 +30,8 @@
             "target_size": (224, 224),
             "crop_area_factor": (0.8, 1.0),
             "aspect_ratio_factor": (3 / 4, 4 / 3),
+            "bounding_box_format": "xywh",
+            "dtype": tf.float32,
```

Comment:
Now that we're using Keras Core, we need to switch to string dtypes instead of `tf.float32`

So for the places in this config where we specify `"dtype": tf.float32` we should instead now specify `"dtype": "float32"`.

After that change I think we can merge this

Reply:
Got it!